src/qp/metrics/point_estimate_metric_classes.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
+from qp.metrics.base_metric_classes import (
+    MetricOuputType,
+    PointToPointMetric,
+)
+
+
+class PointStatsEz(PointToPointMetric):
+    """Copied from PZDC1paper repo. Adapted to remove the cut based on
+    magnitude."""
```

Comment:
I don't think this is the most reasonable docstring, now that we've moved this class out of rail_base.